core/movetree/node.go

```diff
@@ -17,6 +17,10 @@ type GameInfo struct {
 
 	// Initial player turn. This is traditionally the player with the black stones
 	Player color.Color
+
+	// Ranks of both players. Between 1-30k/kyu, 1-9d/dan, or 1-9p/pro
+	WhiteRank string
```

Comment:
Comments should have the format 
"WhiteRank is the rank of the white player."
"blackRank is the rank of the black player."

Try running golint ./...